pkg/options/use_option.go

```diff
@@ -0,0 +1,19 @@
+package options
+
+func (u *UseCmd) Run(ctx *Context) error {
```

Comment:
We could validate if Server/Role exists in the configuration before setting it as default